database/src/main/runs/_add_segmentation_flow.sql

```diff
@@ -0,0 +1,93 @@
+CREATE OR REPLACE FUNCTION runs._add_segmentation_flow(
+    IN  i_segmentation                  HSTORE,
+    IN  i_by_user                       TEXT,
```

Comment:
I would have assumed that input validation would be handled on the service layer.

Reply:
> Thinking of if and how to limit this so not any random person can just type "me", "random", "etc." and spam us, so to speak.
It cannot be a random person calling the functions. There's verified access, particularly `atum_user`. So actually only _atum service_ should have the credentials. And that would be harvesting the data via agent.. "somehow". Or we can completely drop the fields if ensuring some reasonable stuff come it would be a problem.